pysparc/events.py

```diff
@@ -261,3 +262,45 @@ def __init__(self, msg, event_rate=-1):
         traces -= baselines.T
         integrals = [t.compress(t > INTEGRAL_THRESHOLD).sum() for t in traces]
         self.integrals = integrals + [-1, -1]
+
+
+class ConfigEvent(object):
+
+    def __init__(self, master_config):
+        self.pre_coincidence_time = master_config.pre_coincidence_time
+        self.coincidence_time = master_config.coincidence_time
+        self.post_coincidence_time = master_config.post_coincidence_time
+
+        self.gps_latitude = master_config.gps_latitude
+        self.gps_longitude = master_config.gps_longitude
+        self.gps_altitude = master_config.gps_altitude
+
+        self.use_filter = False
+        self.use_filter_threshold = False
+        self.reduce_data = False
+
+        self.mas_version = master_config.version
+        self.mas_ch1_current = master_config.ch1_current
+        self.mas_ch2_current = master_config.ch2_current
+
+        self.mas_ch1_thres_low = -1 * master_config.ch1_threshold_low
+        self.mas_ch1_thres_high = -1 * master_config.ch1_threshold_high
+        self.mas_ch2_thres_low = -1 * master_config.ch2_threshold_low
+        self.mas_ch2_thres_high = -1 * master_config.ch2_threshold_high
```

Comment:
I would prefer the ADC counts, since that makes it easier to use when reconstructing the trigger (no additional conversion to take into account).
We can also make it work that way in the next version of the station-software, though the HiSPARC DAQ sends its values as [mV] (relative to the desired baseline), it also sends its conversion ('mv_to_adc') value and the baseline ('adc_baseline'). That would make it easier to process data without requiring knowledge wether it uses HiSPARC II or III, and the old or new DAQ..